.changeset/eighty-masks-hang.md

```diff
@@ -0,0 +1,6 @@
+---
+'sku': minor
+---
+Add WebP and AVIF image format support to sku
+
+Added support for `webp` and `avif` image formats in, and updated documentation to include these formats in the list of supported image types.
```

Comment:
The docs update don't really need to be mentioned here. I think the first line is great, but would be worth adding an disclaimer about browser support to the changeset directly, as consumers may forget about it if it's not mentioned.